src/components/Calendar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { DatePicker } from 'redux-form-material-ui';
```

Comment:
I think in order to create a good component that deals with Redux-Form 
Material-UI-Redux-Form is a good reference for how to deal with error handling and the input meta props that come by default from Field Component